ui/client/components/modals/CompareVersionsDialog.tsx

```diff
@@ -110,8 +117,11 @@ class VersionsForm extends React.Component<Props, State> {
                     onChange={(e) => this.setState({currentDiffId: e.target.value})}
                   >
                     <option key="" value=""/>
-                    {_.keys(this.state.difference).map((diffId) => (
-                      <option key={diffId} value={diffId}>{diffId}</option>))}
+                    {_.keys(this.state.difference).map((diffId) => {
+                      const isLayoutOnly = this.isLayoutChangeOnly(diffId)
+                      return (
+                        <option key={diffId} value={diffId} disabled={isLayoutOnly}>{diffId} {isLayoutOnly && "(position only)"}</option>)
```

Comment:
Missing translation at "(position only)"?

Reply:
it's hard to introduce i18n here without HOC and i don't like HOCs. i'll wait till rewrite to FC.